models/load-test.go

```diff
@@ -28,6 +28,20 @@ func (l LoadGenerator) Name() string {
 	return string(l)
 }
 
+// SupportedLoadTestMethods - represents the load test method type
+type SupportedLoadTestMethods int
```

Comment:
What factors weigh on a decision to or to not use the SMPS traffic.proto? Would that be a good or bad thing?